app/routes/cdo/edit/change-status.js

```diff
@@ -54,11 +72,16 @@ module.exports = [
     options: {
       auth: { scope: anyLoggedInUser },
       validate: {
-        payload: validatePayload,
+        payload: validateChangeStatusPayload,
         failAction: changeStatusPostFailAction
       },
       handler: async (request, h) => {
         const payload = request.payload
+        const backNav = addBackNavigation(request, false, true)
```

Comment:
Too many params on the call to addBackNav??

Reply:
removed third arg from this and the block above as well.